web/src/frontend/src/digisos/skjema/familie/Barnebidrag.tsx

```diff
@@ -3,13 +3,14 @@ import * as React from "react";
 import SporsmalFaktum from "../../../nav-soknad/faktum/SporsmalFaktum";
 import { radioCheckKeys } from "../../../nav-soknad/utils/faktumUtils";
 import RadioFaktum from "../../../nav-soknad/faktum/RadioFaktum";
+import {LegendTittleStyle} from "../../../nav-soknad/components/sporsmal/Sporsmal";
 
 class Barnebidrag extends React.Component<{}, {}> {
 	render() {
 		const barnebidrag = radioCheckKeys("familie.barn.true.barnebidrag");
 		return (
 			<div className="blokk barnebidrag">
-				<SporsmalFaktum faktumKey={barnebidrag.faktum}>
+				<SporsmalFaktum faktumKey={barnebidrag.faktum} legendTittelStyle={LegendTittleStyle.FET_NORMAL}>
```

Comment:
Kunne vi ha endret dette i noe css for SporsmalFaktum i stedet for å endre det overalt hvor SporsmalFaktum brukes..? Da er det mindre sannsynlig at vi glemmer det noe sted..?

Reply:
Okei, skjønner :)